acceptance/helpers/machine/users.go

```diff
@@ -11,20 +12,25 @@ import (
 )
 
 // CreateEpinioUser creates a new "user" BasicAuth Secret labeled as an Epinio User.
-func (m *Machine) CreateEpinioUser() (string, string) {
+func (m *Machine) CreateEpinioUser(role string, namespaces []string) (string, string) {
 	user, password := catalog.NewUserCredentials()
+	ns := strings.Join(namespaces, "\n")
+
 	secretData := fmt.Sprintf(`apiVersion: v1
 stringData:
   username: "%s"
   password: "%s"
+  namespaces: |
+    %s
```

Comment:
The `Join` at line 17 uses only `"\n"`. That means here that the second and further namespaces will not be properly indented. Join should use `"\n...."` (replace each `.` with space, (*)).

This is likely not realized because the usages I see later do not specify more than one namespace.
Thus not hitting the issue.

(*) I used `.`'s to make the count clear.

Reply:
Ah, yes, good one! Thanks!

Reply:
:smile: 

```
error converting YAML to JSON: yaml: line 8: could not find expected ':'
```

Reply:
How does the formatted YAML look like ?
Hopefully that makes the misformatting obvious.

Reply:
Ah, this was about confirming the issue. I thought the error was happening for my proposed fix.
Agreed that serialization of a properly constructed struct is safer.